metabase/metabase
# Use dedicated endpoint to fetch questions using a model

Makes the model detail page's "Used by" section using a new shape of the `GET /api/card` endpoint to fetch questions using a specific model (added in #26088).

### To Verify

1. Open one of your models from the collection view (or go to `/model/:id/detail`)
2. Ensure "Used by" section lists the question using the model
3. Create a new model and ensure the empty state looks correct
